proto/viam/app/v1/app.proto

```diff
@@ -830,6 +832,41 @@ message CheckPermissionsResponse {
   repeated AuthorizedPermissions authorized_permissions = 1;
 }
 
+message RegistryItem {
+  // The id of the item, containing either:
+  // namespace:item_name when a namespace exists on the org.
+  // org_id:item_name when a namespace does not exist.
+  string item_id = 1;
+  // The id of the organization that owns the item
+  string organization_id = 2;
+  // The public namespace of the organization that owns the module
+  // This is empty if no public namespace is set
+  string public_namespace = 3;
+  // The name of the registry item
+  string name = 4;
+  // The type of the item in the registry
+  viam.app.packages.v1.PackageType type = 5;
+  // The visibility of the registry item
+  Visibility visibility = 6;
+  // The versions of the item that are available
+  repeated VersionHistory versions = 7;
+  // The url to reference for documentation, code, etc.
+  string url = 8;
+  // A short description of the item that explains its purpose
+  string description = 9;
+
+  //For Modules Only
+  // A list of models that are available in the module
+  repeated Model models = 10;
+  // The executable to run to start the module program
+  optional string entrypoint = 11;
```

Comment:
changed this from the [scope](https://docs.google.com/document/d/1-eEz5r2hJL4WaUIVoNAWKKPSTPe6k3VuqUyJP11IXo8/edit) to be optional (since it is for modules only) -- does that seem correct?